src/brs/db/sql/SqlAccountStore.java

```diff
@@ -111,33 +111,16 @@ protected Account load(DSLContext ctx, ResultSet rs) throws SQLException {
 
       @Override
       protected void bulkInsert(DSLContext ctx, ArrayList<Account> accounts) {
-        if ( ctx.fetchExists(ctx.selectOne().from(ACCOUNT).where(ACCOUNT.HEIGHT.eq(Burst.getBlockchain().getHeight())).limit(1)) ) {
```

Comment:
This line was hitting sync performance really hard. It's more efficient to not have this check at all.